lib/toxiproxy_ex/client.ex

```diff
@@ -1,66 +1,59 @@
 defmodule ToxiproxyEx.Client do
   @moduledoc false
 
-  def reset() do
-    client()
-    |> Tesla.post("/reset", %{})
-  end
-
-  def version() do
-    client()
-    |> Tesla.get("/version")
-  end
-
-  def list_proxies() do
-    client()
-    |> Tesla.get("/proxies")
-  end
-
-  def create_proxy(params) do
-    client()
-    |> Tesla.post("/proxies", params)
-  end
-
-  def destroy_proxy(name) do
-    client()
-    |> Tesla.delete("/proxies/#{name}")
-  end
-
-  def enable_proxy(name) do
-    client()
-    |> Tesla.post("/proxies/#{name}", %{enabled: true})
-  end
-
-  def disable_proxy(name) do
-    client()
-    |> Tesla.post("/proxies/#{name}", %{enabled: false})
-  end
-
-  def list_toxics(proxy_name) do
-    client()
-    |> Tesla.get("/proxies/#{proxy_name}/toxics")
-  end
+  alias Tesla.Env
+  alias ToxiproxyEx.ServerError
+
+  # We can remove this once Toxiproxy will fix their stuff. See:
+  # https://github.com/Shopify/toxiproxy/pull/538
+  defmodule BuggyToxiproxyVersionParserMiddleware do
```

Comment:
This could eventually be removed since https://github.com/Shopify/toxiproxy/pull/538 was merged.